templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>What is the cell feature that's being plotted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Since this cell table was featurized by cluster, cell feature refers to cell cluster.  Each block shows the number of cells in each k-Means "topic" cluster that have a particular cell feature.  In the PAP data the cell cluster names showed up on the y-axis but for some reason the cluster IDs showed up here instead.  </p>

Reply:
Okay so for markers this would instead be each channel?

Reply:
Correct

Reply:
Got it. Step 1 is having the cluster names show up here instead of the IDs, which will make it easier to understand. Potentially you could also have the y-axis label change to be clusters if clusters were used, otherwise channels, but I think the main thing is having an interpretable axis label, which changing it should do

Reply:
Should be good now.  The cell cluster names or marker names will now show up on the y-axis tick labels.  The example data doesn't have cluster names so I switched the notebook to use markers instead and it works.